persistence-worker/main.py

```
@@ -83,71 +169,67 @@ async def run(self):
-            print("[INIT] Subscribing to Redis channel: stock:realtime:*")
-            pubsub = redis_client.pubsub()
-            await pubsub.psubscribe("stock:realtime:*")
-            print("[INIT] Subscribed to Redis channel successfully")
+            # PENDING 메시지 먼저 처리
+            await self.process_pending_messages(redis_client, pool)
 
-            # 주기적으로 버퍼를 비우는 태스크
-            asyncio.create_task(self.auto_flush(pool))
-            print("[INIT] Auto-flush task started. Listening for messages...")
+            # 주기적 flush 태스크
+            asyncio.create_task(self.auto_flush(pool, redis_client))
```

Ruff (0.14.11)

Store a reference to the return value of `asyncio.create_task`

(RUF006)